firmware/app/scripts/astyle_format_west_command.py

```
@@ -0,0 +1,61 @@
+# Copyright (c) 2025 Daniel Kampert
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/bin/env python
```

Ruff (0.14.0)

Shebang is present but file is not executable

(EXE001)